core/src/main/java/org/bitcoinj/core/PartiallySignedTransaction.java

```diff
@@ -0,0 +1,542 @@
+/*
+ * Copyright 2019 Giannis L. Jegutanis
```

Comment:
For new files we are using `Copyright by the original author or authors.` to keep things simple/uniform.